script/Common/00_00.txt

```diff
@@ -996,7 +996,7 @@ I tell her about my home―――
 [sha:0b70df24158054c6280e2b77f0a354d23d0b7b18]{
 -- Page 67, Offset 297.
 -- あきはという妹がいて、とてもおとなしくて、いつも僕のあとについてきていたこと。
-I tell her that I have a younger sister named Akiha,
+I tell her that I have a younger sister named Akiha
```

Comment:
might keep

Reply:
idk, pause reads kinda awkward when you see it laid out in game.